buildSrc/build.gradle.kts

```diff
@@ -1,17 +1,62 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+import org.jetbrains.kotlin.gradle.dsl.KotlinVersion
+
 plugins {
+  java
   `kotlin-dsl`
 }
 
 dependencies {
+  // prevent other versions of stdlib from creeping in
+  implementation(libs.kotlinStdlib)
+
   implementation(libs.downloadTaskPlugin)
   implementation(libs.spotlessPlugin)
+  implementation(libs.testloggerPlugin)
+  implementation(libs.detektPlugin)
+  implementation(libs.kotlinPluginKover)
+  implementation(libs.powerassertPlugin)
```

Comment:
kotlin powerassert plugin for better test failure answers:

<img width="874" alt="Screenshot 2024-02-19 at 12 28 33 PM" src="https://github.com/apple/pkl/assets/171897/f926c02c-495f-4437-aee7-36e45b8feb1e">